pkg/core/utils/utils.go

```diff
@@ -63,3 +67,26 @@ func strAt(slice []string, str string) int {
 	}
 	return -1
 }
+
+func SliceContains(in interface{}, elem interface{}) bool {
+	inValue := reflect.ValueOf(in)
+	inType := inValue.Type()
+
+	if inType.Kind() == reflect.Slice {
```

Comment:
Using reflect here looks overkill, the function name has already suggested that the input is a slice.